src/byte_set.rs

```diff
@@ -0,0 +1,237 @@
+use std::{fmt, ops};
+
+/// Compact, allocation-free set of `u8`s.
+#[derive(Copy, Clone, PartialEq, Eq)]
```

Comment:
Should we add Debug traits everywhere? I've been adding them as needed because they sure are handy when debugging.

Reply:
I implemented it manually :)

Reply:
Oh so you did! Very neat :nerd_face: